### PR TITLE
25 site default sidebar contact fragment

### LINKED ISSUE #25: [Site] Default sidebar contact fragment
Main sites (https://www.eder-gmbh.de/unternehmen/ueber-uns/) have a similar default sidebar fragment showing the corporate address & contact details. This should not be maintained on every page.

- [ ] setup fragment doc containing the corporate address & contact details
- [ ] implement block to embed the "contact" fragment - read the contact names from query index or fall back to default corporate address

### PR DESCRIPTION
**Pull request by basecode**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #25 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/das-agratec-landtechnikzentrum-hat-einen-neuen-partner-die-firma-ziegler-cultivation
- After: https://25-site-default-sidebar-contact-fragment--eder-group--hlxsites.hlx.page/newstermine/pressemeldungen/detail/das-agratec-landtechnikzentrum-hat-einen-neuen-partner-die-firma-ziegler-cultivation

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -18,10 +18,14 @@ window.hlx.RUM_GENERATION = 'project-1'; // add your RUM generation information
 function buildHeroBlock(main) {
   const h1 = main.querySelector('h1');
   const picture = main.querySelector('picture');
+  const fullWithSidebarHtml = '<div class="section-metadata"><div><div>style</div><div>Full-width-no-sidebar</div></div></div>';
```
Comment (mhaack) [inline scripts/scripts.js:21]: What is this good for?

File: scripts/scripts.js [hunk 2]
```diff
@@ -18,10 +18,14 @@ window.hlx.RUM_GENERATION = 'project-1'; // add your RUM generation information
...
   // eslint-disable-next-line no-bitwise
   if (h1 && picture && (h1.compareDocumentPosition(picture) & Node.DOCUMENT_POSITION_PRECEDING)) {
     const section = document.createElement('div');
     section.append(buildBlock('hero', { elems: [picture, h1] }));
+
+    // consider a sidebar on every page and automatically set full width no sidebar for hero block.
+    section.insertAdjacentHTML('beforeend', fullWithSidebarHtml);
```
Comment (mhaack) [inline scripts/scripts.js:28]: Every page with no aka default theme has a sidbar. For pages which use the full width aka. no sidewar we set the `wide` template.

File: blocks/contact/contact.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    return '';
+  }
+  return contact.path;
+}
+
+async function getContactHtml(path) {
+  const response = await fetch(`${path}.plain.html`);
+  if (!response.ok) {
+    // eslint-disable-next-line no-console
+    console.log('error loading contact details', response);
```
Comment (mhaack) [inline blocks/contact/contact.js:18]: Please change to error log.

File: blocks/contact/contact.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import { customDecorateIcons, decorateLinkedPictures, getIndex } from '../../scripts/scripts.js';
+
+async function getEmployeeContactPath(name) {
+  const index = await getIndex('contact', '/profiles/query-index.json');
+  const contact = index.find((element) => element.name === name);
+  if (!contact?.name) {
+    // eslint-disable-next-line no-console
+    console.log('contact for name not found', name);
```
Comment (mhaack) [inline blocks/contact/contact.js:8]: Please change to error log.

File: blocks/contact/contact.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  const response = await fetch(`${path}.plain.html`);
+  if (!response.ok) {
+    // eslint-disable-next-line no-console
+    console.log('error loading contact details', response);
+    return '';
+  }
+  const text = await response.text();
+  if (!text) {
+    // eslint-disable-next-line no-console
+    console.log('contact details empty');
```
Comment (mhaack) [inline blocks/contact/contact.js:24]: Please change to error log or maybe remove complete.

File: blocks/contact/contact.css [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  margin: 0;
+  padding: 0;
+}
+
+main .contact .corporate p:first-of-type {
+  text-align: center;
+}
+
+main .contact .employee {
+  border: 1px solid #ddd;
```
Comment (mhaack) [inline blocks/contact/contact.css:19]: Use/add color variable.

File: blocks/contact/contact.css [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  padding: 10px;
+  margin-bottom: 15px;
+  clear: left;
+}
+
+main .contact .employee img {
+  padding: 2px;
+  box-shadow: 0 1px 5px #ddd;
```
Comment (mhaack) [inline blocks/contact/contact.css:27]: Use/add color variable.

File: blocks/contact/contact.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+  block.textContent = '';
+
+  if (employeeNames.length === 0) {
+    const corporateContactHtml = await getContactHtml('/profiles/corporate');
+    await renderContact(corporateContactHtml, block, 'corporate');
+    return;
+  }
+
+  const headline = document.createElement('h1');
+  headline.innerHTML = 'Kontakt';
```
Comment (alexcarol) [inline blocks/contact/contact.js:58]: The original page doesn't seem to say "Kontakt" anywhere, why did you add this title?

Comment (basecode) [reply blocks/contact/contact.js:60]: There are some pages saying "Kontakt" and @mhaack had the idea to add it everywhere "simple and pragmatic".

**Review by alexcarol — APPROVED** [review]
